src/TTextEdit.h

```diff
@@ -125,7 +132,11 @@ public slots:
     bool mInversOn;
     bool mIsDebugConsole;
     bool mIsMiniConsole;
-    bool mIsLowerPane;
+    // Each TConsole instance uses two instances of this class, one above the
+    // other but they need to behave differently in some ways; this, now const,
```

Comment:
Don't need to mention 'now const', I don't think anyone will get confused that it is now const

Reply:
Fair enough.